phases/register.go

```diff
@@ -1,9 +1,119 @@
 package phases
 
-import "fmt"
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
 
-// Register ...
-func Register(progress Progress) error {
-	fmt.Println("Register")
+	"github.com/bitrise-io/go-utils/log"
+)
+
+var baseURL = "https://app.bitrise.io"
+
+var (
+	webhookAttemptCount int
+	webhookAttemptMax   = 3
+)
+
+func startAppRegistration() (string, error) {
+	return "", nil
+}
+
+func registerSSHKey() error {
```

Comment:
we already have these functions in the phases, why do they need a separate function? Register now should have the plain details, phases already ran, just need to have one main logic to do the registration

Reply:
This function is for the actual API call to register the generated SSH key.